bin/dev

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
```

Comment:
shouldn't bin/run also change to reference oclif/core?
https://github.com/oclif/hello-world/blob/main/bin/run